content/authorization/sign-in-with-livechat/index.mdx

```diff
@@ -11,23 +11,33 @@ desc: 'Some apps you build may need to access data from the LiveChat users accou
 
 ## User flow
 
-"Sign in with LiveChat" flow is the easiest way to get access to basic information in LiveChat accounts. It allows you to quickly build an app that can access this information.
+Th **Sign in with LiveChat** flow is the easiest way to get access to basic information in LiveChat accounts. It allows you to quickly build an app that can access that info.
 
-User starts the flow by clicking the "Sign in with LiveChat" button:
+Users start the flow by clicking the **Sign in with LiveChat** button:
 
-![Sign in with LiveChat](images/sign-in-with-livechat-btn.png)
+<!-- <div style="max-width: 250px">
+  <img src="./images/sign-in-with-lc-btn.png" alt="Sign in with LiveChat Button" />
+</div> -->
 
-If user is not signed in to LiveChat, he is asked to do that:
+<button class="lc-btn" type="button" style="margin-right:auto; display:block; margin-bottom: 20px;">
```

Comment:
@jakubsikora, could you help me figure out why this class doesn't make the button blue? Also, I wanted to add a link to Design System to `onlick` on this button, but it didn't work either.